setup.py

```diff
@@ -8,40 +8,12 @@
 from pathlib import Path
 from setuptools import Extension, find_packages, setup
 from setuptools.command.build_ext import build_ext
-
+from intel_extension_for_transformers.tools.utils import get_gpu_family
 
```

Comment:
There is not itrex package in this time.

Reply:
absolute path import: intel_extension_for_transformers/tools/utils.py